tvm-opcodes/src/test/kotlin/org/ton/bytecode/TvmDisasmApiTest.kt

```diff
@@ -28,6 +28,13 @@ class TvmDisasmApiTest {
         assertTrue { MethodId.valueOf(-1) in contractCode.methods }
     }
 
+    @Test
+    fun testWithBadDict() {
```

Comment:
This name is not very descriptive. Could you please rename it somehow and provide some comments?